randovania/gui/ui_files/preset_patcher_energy.ui

```diff
@@ -332,6 +332,44 @@
           </layout>
          </widget>
         </item>
+        <item>
+         <widget class="QGroupBox" name="ingame_difficulty_box">
+          <property name="title">
+           <string>In-Game Difficulty</string>
+          </property>
+          <layout class="QGridLayout" name="gridLayout_5">
+           <item row="0" column="0">
+            <widget class="QLabel" name="ingame_difficulty_label">
+             <property name="text">
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Control how the in-game difficulty options are presented to the player when starting a new Save Slot in the Main Menu:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Normal: &lt;/span&gt;The player is forced to select &amp;quot;Normal&amp;quot; difficulty&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Hard: &lt;/span&gt;The player is forced to select &amp;quot;Hard&amp;quot; difficulty and the seed will be generated using adjusted damage/energy requirements&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Either (Not Recommended): &lt;/span&gt;The player may select either the difficulty. If the player chooses to play on &amp;quot;Hard&amp;quot;, the seed may not be completable because of logic differences&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
What is the purpose of offering the "Either" option in the UI?

I can understand allowing it in the preset file, but in the UI seems like a trap for users?

Reply:
"Either" is the functionality as we have it today. Despite being there from the start, we've only had one or two genuinely impossible seeds reported. The biggest improvement here is changing the default which I'm confident will make this problem never come up again.